### DISCUSSION TITLE
Nested categories

### DISCUSSION THREAD

**Discussion by bdmtra** [engram-design/FeedMe#957] [Q&A]
### Description
I try to import nested categoris. Json is like:
[{"title":"Title""location":[{"Florida":["Key West"]}]}]
I want to be created "Florida" category and nested child "Key West"
But plugin offer me to import that like this:
https://d.pr/free/i/NvgGOm
Only "Key West" category will be created.
Is it possible to import nested categories? What is right json structure for that? 

### Additional info

- Plugin version: 4.1.2
- Craft version: 3.3.18.4

**Comment by redkrupp** [comment]
I've got same question.
